src/clients/java/samples/two-phase/src/main/java/Main.java

```diff
@@ -96,7 +95,6 @@ public static void main(String[] args)
 	    transfers.setCode(1);
 	    transfers.setAmount(500);
 	    transfers.setFlags(TransferFlags.POST_PENDING_TRANSFER);
-	    transfers.getTimestamp();
```

Comment:
Yeah, this one can be removed too. They are no-op.

Reply:
Thanks!